.travis.yml

```diff
@@ -12,7 +12,7 @@ matrix:
     env:
     - COMPILER=g++
     - IMG=gcc-5
-    - CMAKE_EXTRA_FLAGS="-DENABLE_CUDA=Off -DENABLE_COVERAGE=On -DENABLE_C=On"
+    - CMAKE_EXTRA_FLAGS="-DENABLE_CUDA=Off -DCMAKE_BUILD_TYPE=Debug -DENABLE_COVERAGE=On -DENABLE_C=On"
```

Comment:
Can we get Debug builds for gcc8 as well?  We also might want to run with ReleaseWithDebug?

Reply:
Good suggestion. I don't want to blow up the build matrix too much though.

Reply:
Okay.  I'll take whatever I can get..